webhook/webhook.go

```diff
@@ -71,6 +73,13 @@ const (
 	Connect Operation = admissionv1beta1.Connect
 )
 
+var (
+	// GracePeriod is the duration that the webhook will wait after it's
+	// context is cancelled (and probes are failing) before shutting down
+	// the http server.
+	GracePeriod = 30 * time.Second
```

Comment:
Given this is not in data path and it's basically API server calling, should 5-6s suffice?
Also if we inject it, it can be a const :)

Reply:
I made it a var so that folks can customize it from their `main` method as needed, as we do for tests.

When I first added this I thought the tests were hanging 😬